src/components/dashboard/ReceiveToAddress.js

```diff
@@ -10,33 +10,33 @@ import { getDesignRelativeHeight } from '../../lib/utils/sizes'
 import normalize from '../../lib/utils/normalizeText'
 import illustration from '../../assets/Signup/maginLinkIllustration.svg'
 import CopyButton from '../common/buttons/CopyButton'
+import Config from '../../config/config'
 
 export type TypeProps = {
   screenProps: any,
   navigation: any,
   styles: any,
 }
 
-const ReceiveToAddress = ({ screenProps, styles }: TypeProps) => {
-  const { account } = GoodWallet
+const { account } = GoodWallet
+const displayAddress = Config.isTest ? '0x0000000000000000000000000000000000000000' : account
```

Comment:
Please remove this. You should mock GoodWallet.account value in breforeEach and restore in afterEach
each test's stubs/hard-codes/workarounds should be done in the test file

Reply:
Applied fix from #1595